lib/src/main/scala/spinal/lib/bus/regif/BusIfAdapter/AhbLite3BusInterface.scala

```diff
@@ -16,22 +16,35 @@ case class AhbLite3BusInterface(bus: AhbLite3, sizeMap: SizeMapping, regPre: Str
   readError.setAsReg() init False
   readData.setAsReg()  init 0
 
-  val askWrite    = bus.HSEL & bus.HTRANS(1) === True &  bus.HWRITE
-  val askRead     = bus.HSEL & bus.HTRANS(1) === True & !bus.HWRITE
-  val doWrite     = bus.HREADYOUT & RegNext(askWrite, False)
-  val doRead      = bus.HREADYOUT & askRead
+  val askWrite    = bus.HSEL & bus.HTRANS(1) &  bus.HWRITE
+  val askRead     = bus.HSEL & bus.HTRANS(1) & !bus.HWRITE
+  val doWrite     = bus.HREADY & RegNext(askWrite, False)
+  val doRead      = bus.HREADY & askRead
 
   val addressDelay = RegNextWhen(bus.HADDR, askRead | askWrite)
 
-  bus.HREADYOUT := True
   bus.HRESP     := readError
   bus.HRDATA    := readData
```

Comment:
Why not just `val readData = bus.HRDATA`?

Reply:
?? readData is data read from Register need put on bus.HRDATA ?   what is `val readData = bus.HRDATA` ?

Reply:
`val a = b` is just aliasing `a` as another name for `b`. It does not create a new wire.

So you can `a := x` and `y := a` and it is the same as `b := x` and `y := b`.